OneCommunityGlobal/HighestGoodNetworkApp
# Kaikanes lessonform draganddrop

# Description
Added Drag and Drop feature to my lesson from design

## Related PRS (if any):
Added feature to Front end PR
https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/1477

## Main changes explained:
- added the ability to drag and drop

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as anyone
4. go to  http://localhost:3000/bmdashboard/lessonform
5. verify drag and drop functionality works in various view ports and scenarios

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/114305309/009342a7-9157-466f-9d28-0b1bfba1ca0f


## Note:
Still no backend functionality
